lib/active_admin/order_clause.rb

```diff
@@ -29,8 +29,11 @@ def table
     end
 
     def table_column
-      (@column =~ /\./) ? @column :
+      if (@column =~ /\./)
+        @column
```

Comment:
This class was new to me. This is also a worthwhile improvement so thank you. It would be best to resolve the code coverage patch here though before merging. Should be simple enough to do.

I tried out the spec and figured out that the following sample will trigger this branch in the code. Feel free to use this as a starting point for a basic test here for coverage.

```ruby
  describe "" do
    let(:clause) { "posts.id_asc" }
    subject { super().table_column }
    it { is_expected.to eq("posts.id") }
  end
```